ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -0,0 +1,31 @@
+---
+# This task will install the standard actions and packages available in openwhisk-catalog repos.
+
+- set_fact:
+    catalog_location={{ item.value.location }}
+    catalog_repo_url={{ item.value.url }}
+    api_host={{ groups['edge'] | first }}
+
+- set_fact:
+    version="HEAD"
+  when: item.value.version is not defined
+
+- set_fact:
+    version={{ item.value.version }}
+  when: item.value.version is defined
+
+- name: "ensure catalog_location directory exists"
+  file:
+    path: "{{ catalog_location }}"
+    state: directory
+
+- name: download the catalog repository to the catalog location if necessary
+  git:
+    repo: "{{ catalog_repo_url }}"
+    dest: "{{ catalog_location }}"
+    version: "{{ version }}"
+
```

Comment:
I see you changed this to always do the git even if you already have the repo in your workspace.  This solves the issue with Jenkins builds that do not clean up the workspace.  However, as a developer I do not want a git update happening on my catalog repo everytime I do a deploy.  Please create an ansible flag I can use to do an install catalog only (no git clone or update).